src/orchestrator.py

```
@@ -54,18 +55,58 @@ def _get_reports_host_path(self) -> str:
+            config = self.load_config(yaml_file)
+
+            if not config or "tests" not in config:
+                continue  # skip files without tests
+
+            for test_name, test_data in config["tests"].items():
+                original_name = test_name
+
+                # Handle duplicates
+                count = test_name_counts.get(original_name, 0)
+                if count > 0:
+                    warnings.warn(
```

Ruff (0.15.1)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)